src/status/libstatus/chat.nim

```diff
@@ -299,4 +299,11 @@ proc leaveCommunity*(communityId: string) =
 
 proc inviteUserToCommunity*(communityId: string, pubKey: string) =
   let res = callPrivateRPC("inviteUserToCommunity".prefix, %*[communityId, pubKey])#.parseJSON()["result"]
+  debug "RESULT", res
```

Comment:
:hocho:

Reply:
Those are an integral part of the feature SMH

Reply:
ah, makes sense! :100: